ffmpeg script

```diff
@@ -0,0 +1,4 @@
+ffmpeg -i unfragmented.mp4 -g 52 \
```

Comment:
thats the intent, but havent gotten it to work yet. i plan to come back to this later (or someone else can)